README.md

```diff
@@ -271,6 +271,7 @@ boards:
     type: the LAVA device-type of this device
     slave:		(optional) Name of the slave managing this device. Default to first slave found or default_slave if set.
     kvm: (For qemu only) Does the qemu could use KVM (default: no)
+    tap: (For qemu only) Does the qemu could use TAP devices (default: no)
```

Comment:
Why is the default `no`?

Reply:
Basic qemu does not do network. Furthermore the less is mounted in docker the better it is. (security pov).